src/nekontrol/interactive/commands/submit.py

```diff
@@ -82,13 +82,12 @@ def submit(file_path, problem, config):
             )
         problem = file_base
 
-    lang = language.get_lang(file_path)
+    lang = language.get_lang(file_path, config)
```

Comment:
Pass task context perhaps?

Reply:
probably, although thats what it was before this pr ;)

Reply:
actually no, we don't use the runnable part of the language here. we only use it to get the full_name. that's why I want a way to get the higher level type rather than an instance.

Reply:
meaning, there is no task context to pass